internal/compiler/widgets/fluent-base/std-widgets-impl.slint

```diff
@@ -196,13 +197,14 @@ ScrollBar := Rectangle {
 }
 
 export ScrollView := Rectangle {
-    property <length> viewport-width <=> fli.viewport-width;
-    property <length> viewport-height <=> fli.viewport-height;
-    property <length> viewport-x <=> fli.viewport-x;
-    property <length> viewport-y <=> fli.viewport-y;
-    property <length> visible-width <=> fli.width;
-    property <length> visible-height <=> fli.height;
-    property <bool> enabled: true;
+    in-out property <length> viewport-width <=> fli.viewport-width;
+    in-out property <length> viewport-height <=> fli.viewport-height;
+    in-out property <length> viewport-x <=> fli.viewport-x;
+    in-out property <length> viewport-y <=> fli.viewport-y;
+    out property <length> visible-width <=> fli.width;
+    out property <length> visible-height <=> fli.height;
+    in property <bool> enabled: true;
+    // FIXME: internal
```

Comment:
I suggest to enhance the comment by adding where this internal property is used/set.

Reply:
```suggestion
    // FIXME: remove. This property is currently set by the ListView and is used by the native style to draw the scrollbar differently when it has focus
```

Reply:
👍